benchmarks/cpp/nvfuser/dynamic_shape.cpp

```diff
@@ -1,122 +0,0 @@
-#include <torch/csrc/jit/codegen/cuda/executor.h>
```

Comment:
This benchmark was used to estimate heuristic cache latency. Please let me know if you'd like this in a different form.

Reply:
Benchmark is fine, was trying to push it into this PR, instead of the first two: https://github.com/csarofeen/pytorch/pull/1037 looks like it got missed though, but will get it in there.

Reply:
I don't know why this file shows up as deleted in this diff, and not added in https://github.com/csarofeen/pytorch/pull/1037 but that does seem to be what's happening in my local git repo, so maybe something wrong with github not updating.